docs/includes/backup-warning.txt

```diff
@@ -0,0 +1,19 @@
+.. note::
+  When dealing with larger backups, the ``securedrop-admin backup`` command may
+  fail with a ``MemoryError`` at this stage of the operation: "Fetch the backup
+  tarball back to the Admin Workstation".
+  
+  If this happens, a backup was successfully generated, but it is still
+  on the server. Run this command from your ``~/Persistent/securedrop``
+  directory to copy the backup your *Admin Workstation*:
+  
+  .. code:: bash
+
+    rsync -av --progress --partial app:$(ssh app ls -1rt /tmp/sd-backup* | tail -1) install_files/ansible_base/
```

Comment:
is there a reason why it's `ansible_base`? Backups are normally `install_files/ansible-base`

Reply:
That is a typo that I didn't catch. It should be ansible-base.

Reply:
ah good catch, I'm guessing that's a hyphen/underscore typo in the original command

Reply:
It is. You transcribed it correctly.

Reply:
We didn't catch it during the upgrade because we ended up using rsync to transfer the backup back to the app server.

Reply:
(Fixed.)